package.json

```diff
@@ -58,6 +58,7 @@
     "lerna-changelog": "^0.8.2"
   },
   "dependencies": {
+    "camelcase": "^5.0.0",
```

Comment:
not necessary for this PR, but we might want to migrate this and `snake-case` to https://github.com/ember-cli/ember-cli-string-utils, to ensure we use the same rules as Ember itself

Reply:
Thanks for the suggestion, will consider!